### PR TITLE
Updates website

### PR DESCRIPTION
**Pull request by EricWM612**
Adds results table, adds previous games section, cleans code, and some other small things.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,360 +1,529 @@
...
-    <!-- Custom scripts for this template -->
-    <script src="js/freelancer.min.js"></script>
+  <!-- Custom scripts for this template -->
+  <script src="js/freelancer.min.js"></script>
 
-  </body>
+</body>
 
-</html>
+</html>
```
Comment (topoftheyear) [inline docs/index.html]: Please do not remove the newline at the end of this file.

Comment (EricWM612) [reply docs/index.html]: VSCode decided it wasn't necessary, along with the indentation within<html>, and who am I to argue with that?

Comment (topoftheyear) [reply docs/index.html]: Then tell VSCode to argue with this old man from MIT who's worked at Dow Jones for 24 years https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file?share=1

File: docs/index.html [hunk 2]
```diff
@@ -1,360 +1,529 @@
...
 
-    <!-- Custom scripts for this template -->
-    <script src="js/freelancer.min.js"></script>
+  <!-- Custom scripts for this template -->
+  <script src="js/freelancer.min.js"></script>
 
-  </body>
+</body>
 
 </html>
```
Comment (topoftheyear) [inline docs/index.html]: You've appeased the old man.

**Pull request merged**